actioncable/Rakefile

```diff
@@ -1,15 +1,14 @@
 require 'rake/testtask'
 require 'pathname'
-require 'sprockets'
-require 'coffee-script'
 require 'action_cable'
+require 'blade'
 
 dir = File.dirname(__FILE__)
 
 task :default => :test
 
 task :package => "assets:compile"
-task "package:clean" => "assets:clean"
+task "package:clean"
```

Comment:
All the other gems define an empty "package:clean" task. Introduced in d6f2000a67cc63aa67414c75ce77de671824ec52. I think Action Cable was the only one to implement it. Can probably remove them all now. @matthewd?